### PR TITLE
Reset sort order on set_view_type()

### LINKED ISSUE #16: Reversed polarity for progress bars
The progress bar indicator is reversed

Here is a screenshot
http://s4.postimg.org/o19x16p65/kde_polarity.png

### PR DESCRIPTION
**Pull request by bp0**
Required to fix #16, for me.

### CODE REVIEW

File: shell/shell.c [hunk 1]
```diff
@@ -897,6 +897,7 @@ static void set_view_type(ShellViewType viewtype, gboolean reload)
 
     shell->normalize_percentage = TRUE;
     shell->view_type = viewtype;
+    shell->_order_type = 0;
```
Comment (lpereira) [inline shell/shell.c]: I'm curious why this fixes the issue: this is setting the order type to descending.

Comment (bp0) [inline shell/shell.c]: The % is inverted later if shell->_order_type = 1.
https://github.com/bp0/hardinfo/blob/2e585991f0880a8a6596c65a364c622d28b24961/shell/shell.c#L1245

Comment (bp0) [inline shell/shell.c]: The benchmarks each specifically set OrderType in the shell params. Filesystems does not, so it ends up using whatever the last one was. Resetting to 0 (descending) and having that as a default seems like the right fix, considering that 1 (ascending) has that special handling.

Comment (lpereira) [inline shell/shell.c]: Alright, makes sense. Please just use the enum value instead of 0 for clarity.

**Comment by lpereira** [comment]
Squashed and merged, thanks! (For small fixups such as these, please use interactive rebase, squash/fixup, and force push to the branch the PR has been opened. This makes my life a little bit easier and reduces noise in the git history.)

**Pull request merged**